### PR TITLE
update Json Deserialize

### PR DESCRIPTION
**Pull request by cloud8little**

### CODE REVIEW

**Comment by erikzhang** [comment]
Update the dependency?

**Review by erikzhang — APPROVED** [review]

File: csharp/Oracle/Oracle.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo.SmartContract.Framework" Version="3.0.0-preview5" />
+    <PackageReference Include="Neo.SmartContract.Framework" Version="3.0.0-rc1" />
```
Comment (shargon) [inline csharp/Oracle/Oracle.csproj:10]: I prefer to use the last one

Comment (erikzhang) [reply csharp/Oracle/Oracle.csproj:10]: `3.0.0-rc1` is the last one, right?

Comment (shargon) [reply csharp/Oracle/Oracle.csproj:10]: `3.0.0-CI00251` ?

Comment (erikzhang) [reply csharp/Oracle/Oracle.csproj:10]: I think the examples should be more stable. It is difficult for normal users to use the packages on myget.

**Review by shargon — APPROVED** [review]

**Pull request merged**